Realm/RLMObjectStore.mm

```diff
@@ -91,6 +91,23 @@ static inline void RLMVerifyInWriteTransaction(__unsafe_unretained RLMRealm *con
     }
 }
 
+static inline RLMObjectSchema *RLMGetVerifiedObjectSchema(__unsafe_unretained RLMRealm *const realm,
+                                                          NSString *const objectClassName) {
+    RLMObjectSchema *schema = [realm.schema schemaForClassName:objectClassName];
+    if (!schema) {
+        @throw RLMException(@"Object type '%@' is not persisted in the Realm. "
```

Comment:
The same as in https://github.com/realm/realm-cocoa/pull/3651#discussion_r64659667 should apply here. This PR should just do functional changes in isolation.